pyaedt/circuit.py

```diff
@@ -181,35 +153,69 @@ def from_rkm_to_aedt(code):
 
 
 class Circuit(FieldAnalysisCircuit, object):
-    """Circuit object.
+    """Circuit interface.
```

Comment:
I thought we were going to use this convention for the short summary of a class: """Circuit class."""  I've been making this change. How would I know that this is an interface?

Reply:
`Circuit interface class` is perhaps the best.  Reason being we're providing a class to interface with the AEDT circuit model.

One of the biggest problems with our existing docs is that the classes can't stand on their own.  We need one sentence that fully describes a class and its usage.  Just saying it's a circuit class to the user will be "well duh...", but what is it?  What is it used for?